build.gradle

```diff
@@ -0,0 +1,203 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id 'java'
+    id 'maven-publish'
+}
+
+repositories {
+    mavenLocal()
+    maven {
+        url = uri('https://repo.maven.apache.org/maven2/')
+    }
+}
+
+dependencies {
+    compileOnly 'org.projectlombok:lombok:1.18.20'
+    annotationProcessor 'org.projectlombok:lombok:1.18.20'
+    testCompileOnly 'org.projectlombok:lombok:1.18.20'
+    testAnnotationProcessor 'org.projectlombok:lombok:1.18.20'
+    implementation 'com.browserstack:browserstack-local-java:1.0.3'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.12.2'
+    implementation 'com.fasterxml.jackson.core:jackson-databind:2.12.2'
+    implementation 'org.apache.commons:commons-lang3:3.12.0'
+    implementation 'junit:junit:4.12'
+    implementation 'org.seleniumhq.selenium:selenium-java:3.141.59'
+    implementation 'ch.qos.logback:logback-classic:1.2.3'
+    implementation 'org.seleniumhq.selenium:selenium-api:3.141.59'
+    implementation 'io.qameta.allure:allure-junit4:2.13.8'
+    implementation group: 'org.apache.maven.plugins', name: 'maven-surefire-plugin', version: '2.21.0'
```

Comment:
please change it to above dependecy format.

Reply:
Done